frontend/src/actions/config.ts

```diff
@@ -0,0 +1,2 @@
+export const url = 'https://ervin-els-backend.herokuapp.com/api';
+// export const url = '127.0.0.1:8000/api';
```

Comment:
Since you have a local and dev server, you can shorten this one using ternary and a another .env variable to determine if it is local or dev.
and add also this env to the dev server(heroku)

1. Create a variable in env
>    LARAVEL_APP_ENV='local' // 'dev' if in dev server
  
2. Then call the .env variable and make a condition
>   export const url = process.env.LARAVEL_APP_ENV == 'local' ? '127.0.0.1:8000/api' : 'https://ervin-els-backend.herokuapp.com/api';